docs/_includes/build-sidebar.html

```diff
@@ -1,9 +1,10 @@
 {% if page.author_profile or layout.author_profile or page.sidebar %}
   <div class="sidebar sticky">
   {% if page.author_profile or layout.author_profile %}{% include author-profile.html %}{% endif %}
-  {% if page.editor_profile or layout.editor_proflie %}
-    <hr/>
-    {% include editor-profile.html %}
+  {% if page.editor_profile or layout.editor_profile %}
+  {% if page.author == page.editor %}{% else %}
```

Comment:
Prefer `{% if page.author != page.editor %}`

Reply:
done ... wasn't too sure about the syntax during testing. thanks for the guidiance